float64 mainPos
float64 jibPos
